akka-actor/src/main/scala/akka/actor/Actor.scala

```diff
@@ -272,6 +272,7 @@ class ActorInterruptedException private[akka] (cause: Throwable) extends AkkaExc
  */
 @SerialVersionUID(1L)
 final case class UnhandledMessage(@BeanProperty message: Any, @BeanProperty sender: ActorRef, @BeanProperty recipient: ActorRef)
+  extends NoSerializationVerificationNeeded
```

Comment:
I noticed that if a message that is not serializable is published as unhandled then the unhandled message is reported as not serializable and hiding the root cause